ppr-ui/src/views/Search.vue

```diff
@@ -273,12 +274,13 @@ export default class Search extends Vue {
       if (this.selectedMatches?.length >= 75) {
         shouldCallback = true
       }
-      const statusCode = await submitSelected(this.getSearchResults.searchId, this.selectedMatches, shouldCallback)
+      let statusCode = await submitSelected(this.getSearchResults.searchId, this.selectedMatches, shouldCallback)
       this.loading = false
+      statusCode = 590
```

Comment:
hey steph, I think you forgot to take this line out haha

Reply:
whoops